i18n/en.json

```diff
@@ -303,6 +303,10 @@
   "admin.billing.subscription.cloudTrial.subscribeButton": "Upgrade Now",
   "admin.billing.subscription.cloudTrialBadge.daysLeftOnTrial": "{daysLeftOnTrial} trial days left",
   "admin.billing.subscription.cloudYearlyBadge": "Yearly",
+  "admin.billing.subscription.complianceScreenFailed.button": "Continue with free version",
+  "admin.billing.subscription.complianceScreenFailed.subtitle": "We are processing your cloud upgrade and will get back to you within 3 business days. In the meantime, please continue to enjoy the free version.",
```

Comment:
> Good stuff @nickmisasi -- let's re-use the copy from here: https://mattermost.atlassian.net/browse/MM-49496?focusedCommentId=140976 to keep things consistent

Bringing this here so it's in-context @nickmisasi

Reply:
`Your transaction is being reviewed`

`We will check things on our side and get back to you within 3 days once your Cloud Subscription upgrade is approved. In the meantime, please feel free to continue using the free version of our product.`

What you linked mentions licenses. Does `Cloud Subscription upgrade` in place of `license` work?

Is the CTA ok?

Reply:
* Cloud subscription upgrade

CTA: Continue with Cloud Free

Reply:
@johndavidlugtu all set
![image](https://user-images.githubusercontent.com/12176405/216411061-d67c4dc0-2c1f-48de-9748-3bcc60ebd682.png)